Help/Hackathon.rst

```diff
@@ -0,0 +1,196 @@
+go from "low" to "high". Of course, for this hackathon, you will choose a
+"low". In order to do so, visit coala.io/low and take a few moments to look
+through the issues. You should, of course, try to choose one that has noone
+assigned (even though in the past someone worked on it, if nobody is assigned,
+feel free to take it).
+
+.. note::
+
+  Don't work at an issue without being assigned. To get assigned, simply
+  leave a comment meaning that you would like to work on it, and in the
+  shortest time a maintainer will assign you to it. However, since our
+  resources are limited, if you comment first that you want to work on it, it
+  should be enough, and if you see someone that has commented faster than you
+  on an issue you found (and the person is from the hackathon), imagine that
+  they are already working on it.
+
+If you've made it this far, you already have an issue assigned to you.
+Now you gotta work on the issue. In order to do so, you should check which
+repository the issue you want to work on belongs to, from the 3 repos listed
+above. Simply clone it using:
+
+::
+
+  git clone https://github.com/coala/coala-bears
+
+or the correct link corresponding to the repository. Now you'll need an online
+copy of the repo (which is called a fork) which can be done by going on the
+repository page and clicking on the ``Fork`` button on the top right of the
+page.
+
+Fixing the issue
+~~~~~~~~~~~~~~~~
+
+Now you're at the hardest part. You need to fix that issue in the code
+you have locally. This part cannot be contained in a tutorial as it differs
+from issue to issue. Mostly, on low difficulty issues, enough information
+should be given in the description or the comments of the issue. If not,
+the coala maintainers present at the hackathon should probably help you as
+it's your duty to ask them.
+
+Testing your Changes
+~~~~~~~~~~~~~~~~~~~~
+
+If you have done the correct changes, you should test it. If you are working
+in the ``documentation`` repository, there's nothing to be tested, but if
+you aren't, you should take a look at
+`this link <https://api.coala.io/en/latest/Developers/Executing_Tests.html>`__
+as it shortly describes how to run our tests.
+
+Getting the Code in the Master Branch
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now we need to create a commit. To do so, type ``git status`` and note that
+the files you changed are written in red. If so, add them using ``git add``.
+Once you have added all the files, type ``git commit``. This will open up
+``vim`` or ``nano`` and ask you to write a commit message. In order to do so,
+please read coala.io/commit, which shortly describes how to write a smart
+commit message. After doing that, save it.
+
+You cannot push to the original repository as you do not have the rights.
+You will, instead, push to the fork you have created. To do that, you have
+to add it locally by typing ``git remote add myfork fork_link``, where myfork
```

Comment:
nah, i feel like its where it should be, this tutorial is not meant to teach them git, if they add the remote here it will be fine :D